cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -83,19 +80,21 @@
                 {% if show_categories %}
                     {% import 'macros/category-slug.html' as category_slug %}
                     {% if cat_controls and 'newsroom' in cat_controls.page_type and is_blog(post) %}
-                        {{ category_slug.render('blog', page_url, '', false) }}
+                        {{ category_slug.render(category='blog', href=url) }}
                     {% else %}
                         {% for cat in post.categories.all() %}
                             {% if loop.index > 1 %}
                                 |
                             {% endif %}
-                            {{ category_slug.render(cat.name, page_url, '', false) }}
+                            {{ category_slug.render(category=cat.name, href=url) }}
                         {% endfor %}
                     {% endif %}
                 {% endif %}
             </div>
         </div>
-         {% if 'EventPage' in post.specific_class.__name__ %}
+        {% cache post.post_preview_cache_key %}
```

Comment:
This section above line 95 (the beginning of the cache) will no longer be cached, thus fixing the issue with the wrong icons displaying (determined by the category_slug macro).

Reply:
Will this require some kind of manual step to clear the entire cache when this goes out? Let's say before the fix some page with primary key 123 has the existing section cached (the whole `render` macro) under cache key `post_preview_123`. After this change, the next time the page is rendered, the template will insert that cached content between the newly moved `{% cache %}{% endcache %}` tags. Will that make a double header appear -- first the new, non-cached one, and then a second one that was in the cache?

Reply:
Yes, exactly -- was planning to clear the cache as soon as this is deployed.